Packs/AzureStorage/Integrations/AzureStorage/README.md

```diff
@@ -0,0 +1,637 @@
+Deploy and manage storage accounts and blob services.
+This integration was integrated and tested with version 2019-06-01 of Azure Storage
+## Permissions
```

Comment:
Please fill this section with the required permissions if necessary.

Reply:
I added here the authentication, as Arseny said.